Cargo.toml

```diff
@@ -11,14 +11,14 @@ version = "0.1.6"
 readme = "README.md"
 
 [dependencies]
-env_logger = { version = "0.9", default-features = false }
-log = "0.4"
-object = { version = "0.26", default-features = false, features = ["read_core", "elf", "std"] }
+env_logger = { version = "0.10", default-features = false }
 getrandom = "0.2"
+log = "0.4"
+object = { version = "0.30", default-features = false, features = ["read_core", "elf", "std"] }
 
 [dev-dependencies]
 assert_cmd = "2.0"
-rstest = "0.11"
+rstest = { version = "0.16", default-features = false }
```

Comment:
Why deactivating default features? Is it good practice for compilation size?

Reply:
I wouldn't say that one should do it per default, but quite often, it can significantly reduce the number of sub-dependencies. Also, it mainly benefits compilation time and not necessarily compilation size.

`rstest v0.13` added a dependency on `futures`, which pulls in many `async` related stuff. Since we don't use `rstest` for `async` functions, we can eliminate these dependencies by disabling the default features.

Reply:
Great I will use this trick more often!